pkg/hintrunner/zero/zerohint.go

```diff
@@ -131,6 +131,8 @@ func GetHintFromCode(program *zero.ZeroProgram, rawHint zero.Hint, hintPC uint64
 		return createFastEcAddAssignNewXHinter(resolver)
 	case ecDoubleSlopeV1Code:
 		return createEcDoubleSlopeV1Hinter(resolver)
+	case reduceV1Code:
+		return createReduceV1(resolver)
```

Comment:
You need to update this to `createReduceV1Hinter`

Try running `make unit` to make sure everything runs.